src/constants.ts

```diff
@@ -62,16 +61,16 @@ export const CLASSES = {
 };
 
 // 1-based in case we ever do `if(element.fontFamily)`
-export const FONT_FAMILY = {
-  1: "Virgil",
-  2: "Helvetica",
-  3: "Cascadia",
-} as const;
+export enum FONT_FAMILY {
+  Virgil = "1",
+  Helvetica = "2",
+  Cascadia = "3",
+}
```

Comment:
let's keep using numbers here so we don't have to unnecessarily migrate to strings